### PR TITLE
Add release notes for e5.2.1

### PR DESCRIPTION
**Pull request by yanzhiemq**

### CODE REVIEW

File: en_US/changes/changes-ee-5.2.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Releases
 
+
+## e5.2.1
+
+### Enhancements
+
+- [#11568](https://github.com/emqx/emqx/pull/11568) Added support for defining templates for MQTT publish properties in Republish rule action.
```
Comment (wivwiv) [inline en_US/changes/changes-ee-5.2.md]:
```suggestion
- [#11568](https://github.com/emqx/emqx/pull/11568) Added support for defining templates for MQTT 5.0 publish properties and user properties in Republish rule action.
```

File: zh_CN/changes/changes-ee-5.2.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 # 版本发布
 
+## e5.2.1
+
+### 增强
+
+- [#11568](https://github.com/emqx/emqx/pull/11568) 在消息重发布规则动作中，支持为 MQTT 发布属性设定模板。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.2.md]:
```suggestion
- [#11568](https://github.com/emqx/emqx/pull/11568) 在消息重发布规则动作中，支持设置 MQTT 5.0 发布属性与用户属性。
```

File: zh_CN/changes/changes-ee-5.2.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+- [#11612](https://github.com/emqx/emqx/pull/11612) 在节点疏散期间，疏散所有断开连接的会话，而不仅仅是那些以 `clean_start` 设置为 `false` 开始的会话。
+
+### 修复
+
+- [#11568](https://github.com/emqx/emqx/pull/11568) 修复了一个未明确定义的内置规则动作配置，以避免该配置被理解为自定义用户函数。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.2.md:20]: The pr and publish properties is the same, wrong URL?

**Review by wivwiv — CHANGES_REQUESTED** [review]

File: en_US/changes/changes-ee-5.2.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+
+### Enhancements
+
+- [#11568](https://github.com/emqx/emqx/pull/11568) Added support for defining templates for MQTT 5.0 publish properties and user properties in Republish rule action.
+
+- [#11612](https://github.com/emqx/emqx/pull/11612) During node evacuation, evacuate all disconnected sessions, not only those started with `clean_start` set to `false`.
+
+### Bug Fixes
+
+- [#11568](https://github.com/emqx/emqx/pull/11568) Fixed an issue where an ill-defined built-in rule action config could be interpreted as a custom user function.
```
Comment (id) [inline en_US/changes/changes-ee-5.2.md:30]: Should we add this one? https://github.com/emqx/emqx/pull/11630

Comment (yanzhiemq) [inline en_US/changes/changes-ee-5.2.md:30]:
```suggestion
- [#11568](https://github.com/emqx/emqx/pull/11568) Fixed an issue where an ill-defined built-in rule action config could be interpreted as a custom user function.
- [#11630](https://github.com/emqx/emqx/pull/11630) Fixed an issue when core node could be stuck at `mria_schema:bootstrap/0`.
```

Comment (yanzhiemq) [reply en_US/changes/changes-ee-5.2.md:30]: Done, @Meggielqk please help to add translated version

File: zh_CN/changes/changes-ee-5.2.md [hunk 5]
```diff
@@ -4,12 +4,27 @@
 
 ### 增强
 
+- [#11487](https://github.com/emqx/emqx/pull/11487) 将 bcrypt 的工作因子 (work factor) 限制在5-10的范围内，因为较高的值会消耗太多 CPU 资源。Bcrypt 库已更新以允许并行哈希计算。
 - [#11568](https://github.com/emqx/emqx/pull/11568) 在消息重发布规则动作中，支持设置 MQTT 5.0 发布属性与用户属性。
 - [#11612](https://github.com/emqx/emqx/pull/11612) 在节点疏散期间，疏散所有断开连接的会话，而不仅仅是那些以 `clean_start` 设置为 `false` 开始的会话。
+- [#11532](https://github.com/emqx/emqx/pull/11532) 改进了对解析无效数据包时的错误消息，以提供更清晰的错误提示。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.2.md]:
```suggestion
- [#11532](https://github.com/emqx/emqx/pull/11532) 改进了解析无效数据包时的错误消息，以提供更清晰的错误提示。
```

File: zh_CN/changes/changes-ee-5.2.md [hunk 6]
```diff
@@ -4,12 +4,27 @@
...
 
 ### 修复
 
+- [#11493](https://github.com/emqx/emqx/pull/11493) 修复了关于 `/api/v5/publish` 的示例和文档中有关错误请求响应的问题。之前的文档示例指出错误请求的响应可以在响应体中返回一个列表，但实际情况并非如此。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.2.md]:
```suggestion
- [#11493](https://github.com/emqx/emqx/pull/11493) 修复了关于 `/api/v5/publish` 的 REST API 示例文档中错误请求响应的问题。之前的文档示例指出错误请求的响应可以在响应体中返回一个列表，但实际情况并非如此。
```

File: zh_CN/changes/changes-ee-5.2.md [hunk 7]
```diff
@@ -4,12 +4,27 @@
...
+- [#11499](https://github.com/emqx/emqx/pull/11499) 升级 Erlang/OTP 至 25.3.2-2，此版本从 mnesia_hook 日志消息中排除了敏感数据。
+- [#11506](https://github.com/emqx/emqx/pull/11506) 空的跟踪日志文件将不再被下载。在实施此修复后，尝试使用 GET 请求 `/api/v5/trace/clientempty/download` 下载空的跟踪日志文件时，服务器现在将返回 404 状态码以及以下 JSON 消息：`{"code":"NOT_FOUND","message":"Trace is empty"}`。如果在日志文件中未找到与跟踪条件匹配的事件，将触发此响应。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.2.md]:
```suggestion
- [#11506](https://github.com/emqx/emqx/pull/11506) 此前尝试下载不存在的跟踪日志文件时，会下载一个空的文件。在实施此修复后，尝试使用 GET 请求 `/api/v5/trace/clientempty/download` 下载不存在的跟踪日志文件时，服务器现在将返回 404 状态码以及以下 JSON 消息：`{"code":"NOT_FOUND","message":"Trace is empty"}`。
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

**Review by Meggielqk — APPROVED** [review]

**Review by wivwiv — APPROVED** [review]

**Pull request merged**